include/deal.II/numerics/data_postprocessor.h

```diff
@@ -104,11 +102,11 @@ namespace DataPostprocessorInputs
    * as a template parameter, the DoFHandler type to which the cell that
    * is currently being processed belongs. This is knowledge you typically
    * have in an application: for example, if your application runs in
-   * @p dim space dimensions, uses a hp::DoFHandler, and you are currently
-   * using the DataOut class, then the cells that are worked on have data
-   * type <code>DataOut<dim>::cell_iterator</code>. Consequently, in a
+   * @p dim space dimensions and you are currently using the DataOut class,
+   * then the cells that are worked on have data type
+   * <code>DataOut<dim>::cell_iterator</code>. Consequently, in a
    * postprocessor, you can call
-   * <code>inputs.get_cell@<hp::DoFHandler@<dim@> @> </code>. For technical
+   * <code>inputs.get_cell@<DoFHandler@<dim@> @> </code>. For technical
```

Comment:
Let's make `DoFHandler` the default template argument:
```suggestion
   * <code>inputs.get_cell()</code>. For technical
```

Reply:
Do you mean in `get_cell` and `set_cell`?

I don't understand what you want to add - we need a template since we do not know dim and spacedim of the DoFHandler. Do you want to do
```cpp
template <int dim, int spacedim = dim>
void
set_cell(const typename DoFHandler<dim, spacedim>::cell_iterator &cell);
```
?

Reply:
I see. Then let's keep it like this!